site/soc/software/config/versions.yaml

```diff
@@ -179,6 +179,7 @@ data:
       libvirt:
         libvirt: "{{ suse_osh_registry_location }}/openstackhelm/libvirt:{{ suse_infra_image_version }}"
       mariadb:
+        mariadb: "registry.suse.de/devel/cloud/10/containers/containers/soc/mariadb:10.2.25"
```

Comment:
out of scope for this PR - but are there plans to make the containers available on OBS? I think we want to also use theses containers for upstream CI.

Reply:
Yes (in SOC-10393).